redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -69,31 +73,30 @@ public function asImage()
         $this->asImage = true;
 
         $this->image = [];
-        $this->image['format'] = strtolower(rex_file::extension($this->getMediapath()));
         $this->image['src'] = false;
 
         // if mimetype detected and in imagemap -> change format
         if (class_exists('finfo') && $finfo = new finfo(FILEINFO_MIME_TYPE)) {
             if ($ftype = @$finfo->file($this->image['filepath'])) {
                 if (array_key_exists($ftype, $this->mimetypeMap)) {
-                    $this->image['format'] = $this->mimetypeMap[$ftype];
+                    $this->format = $this->mimetypeMap[$ftype];
                 }
             }
         }
 
-        if ($this->image['format'] == 'jpg' || $this->image['format'] == 'jpeg') {
-            $this->image['format'] = 'jpeg';
+        if ($this->format == 'jpg' || $this->format == 'jpeg') {
```

Comment:
vllt können wir das `$this->image['format']` ->`$this->format` refactoring in einem separaten PR vorher landen, damit dieser hier übersichtlicher wird?